pyproject.toml

```
@@ -10,9 +10,12 @@ authors = [
 ]
 license = { file = "LICENSE" }
 readme = { file = "README.md", content-type = "text/markdown" }
-requires-python = ">=3.11"
+requires-python = ">=3.11,<3.13"
 dependencies = [
     "boto3",
+    "spacy",
```

@bamader what do you think about just putting the dependencies here, so the installation workflow is the same for both bootstrap.sh and the Dockerfile?
```suggestion
    "spacy >=3.8, <3.9",
    "en_core_web_sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.8.0/en_core_web_sm-3.8.0.tar.gz",
    "spacy-lookups-data",
```